packages/discord.js/typings/index.d.ts

```diff
@@ -5433,7 +5429,7 @@ export type VoiceBasedChannel = Extract<Channel, { bitrate: number }>;
 
 export type GuildBasedChannel = Extract<Channel, { guild: Guild }>;
 
-export type NonCategoryGuildBasedChannel = Exclude<GuildBasedChannel, CategoryChannel>;
+export type CategoryChildChannel = Exclude<Extract<Channel, { parent: CategoryChannel | null }>, CategoryChannel>;
```

Comment:
Is the Exclude necessary?

Reply:
Yes otherwise the type would include CategoryChannel. The property is inherited from GuildChannel and even if you override it with null (which is the value it will always have) it would still show up, I tested that